lib/CL/devices/level0/level0-driver.cc

```diff
@@ -2297,7 +2297,16 @@ bool Level0Device::setupDeviceProperties(bool HasIPVersionExt) {
     ClDev->num_serialize_entries = 2;
     ClDev->serialize_entries = LEVEL0_SERIALIZE_ENTRIES;
     ClDev->llvm_cpu = nullptr;
+#ifdef USE_LLVM_SPIRV_TARGET
+#ifdef ENABLE_LEVEL0_EXTRA_FEATURES
+    ClDev->llvm_target_triplet = "spirv64v1.3-unknown-unknown";
+#else
+    ClDev->llvm_target_triplet = "spirv64v1.2-unknown-unknown";
```

Comment:
Probably should consult the level0 driver and set SPIR-V triplet supported by the driver.